ts-server/.editorconfig

```diff
@@ -0,0 +1,21 @@
+; top-most EditorConfig file
+root = true
+
+[*]
+indent_style = tab
```

Comment:
For [accessibility reasons](https://alexandersandberg.com/tabs-for-accessibility/)